examples/opentelemetry/docker-compose.yaml

```diff
@@ -63,9 +63,11 @@ services:
     healthcheck:
       test: ["CMD-SHELL", "curl -sf http://localhost:13133 || exit 1"]
       interval: 1s
-      timeout: 120s
+      timeout: 10s
```

Comment:
im not sure why this change is here or whether its desirable - i think it would make it retry every 1/12s

that said - 120 retries is probably too high - not sure where these values came from originally

Reply:
The collector comes up in a second or two tops. 120s is IMO just cause for FUD for the end user.

Reply:
cool - probably in that case tho - my reading of docs (maybe incorrectly) is that we would want also to change the `retries` values also to prevent it spinning out with healthchecks - happy to be wrong about that

Reply:
ok - nm - i misread - `interval` is set

in which case i think `retries` is most likely not required at all, but im possibly missing something there

Reply:
> Interval
> 
> The health check will first run interval seconds after the container is started, and then again interval seconds after each previous check completes.
> 
> Timeout
> 
> If a single run of the check takes longer than timeout seconds then the check is considered to have failed.
> 
> Retries
> 
> It takes retries consecutive failures of the health check for the container to be considered unhealthy.

From [StackOverflow](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://stackoverflow.com/questions/68186759/docker-compose-healthcheck-options&ved=2ahUKEwjXqrbV_-6FAxXqVfEDHThtD_0QFnoECB4QAQ&usg=AOvVaw3Jka0tjOljGCdCNLzBOugc) and [this cheatsheet](https://kapeli.com/cheat_sheets/Dockerfile.docset/Contents/Resources/Documents/index). I feel retries are useless. The collector will virtually always answer if it's alive (unless under the most massive GC I ever saw it doing), and if it's not alive, it's 99% a config issue, which are not helped by retries.

Reply:
thanks - i didnt read docs properly obviously - not sure in that case but what we want to avoid at all costs is making the sandbox flakey in ci

Reply:
I don't feel getting rid of retries in this case should make anything flakier tbh. If anything, it'd fail faster.